helpdesk/templates/helpdesk/ticket_list.html

```diff
@@ -415,6 +422,27 @@ <h5 class="mb-0">
                         }
                     },
                     {data: "submitter"},
+                    {
+                        data: "last_followup",
+                        render: function (data, type, row) {
+                            let locale = window.helpdesk_settings.LANGUAGE_CODE; 
```

Comment:
Change this to:
let locale = navigator.language || navigator.userLanguage || window.helpdesk_settings.LANGUAGE_CODE;

It will allow rendering of the timestamp in the users browser locale which will generally be wherever the user is located in the world.